src/pages/HappDetails/HappDetails.test.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react'
+import { render, fireEvent, act } from '@testing-library/react'
+import { Router } from 'react-router-dom'
+import { createMemoryHistory } from 'history'
+import { MockedProvider } from '@apollo/react-testing'
+import wait from 'waait'
+import HappDetails from './HappDetails'
+import { appOne as appHoloFuel } from 'mock-dnas/happStore'
+import AllAvailableHappsQuery from 'graphql/AllAvailableHappsQuery.gql'
+import EnableHappMutation from 'graphql/EnableHappMutation.gql'
+import DisableHappMutation from 'graphql/DisableHappMutation.gql'
+
+const enableHappMock = {
```

Comment:
wondering about just importing all these mocks from `BrowseHapps.test.js` It's probably not right, just a thought.

Reply:
Ideally this would get extracted somewhere. Pulling one test from another seems like a bad idea. :P

Reply:
I'd leave it as it is right now. If there are more cases like this, perhaps we can just setup some space in project for test related utils etc.